indico/htdocs/js/indico/jquery/ajaxqbubble.js

```diff
@@ -0,0 +1,102 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget('indico.ajaxqbubble', $.indico.qbubble, {
+        options: {
+            onHide: null,
+            ajax: {
+                url: null,
+                method: 'GET',
+                cache: false,
+                success: null
+            },
+            position: {
+                effect: false
+            },
+            content: {
+                text: $T.gettext('Loading...')
+            }
+        },
+
+        _create: function() {
+            var self = this;
+            var ajaxOptions = _.pick(self.options, 'ajax').ajax;
+
+            self.options = $.extend(true, {}, _.omit(self.options, 'ajax'), {
+                events: {
+                    show: function(evt, api) {
+                        $.ajax($.extend({}, ajaxOptions, {
+                            url: ajaxOptions.url,
+                            method: ajaxOptions.method,
+                            cache: ajaxOptions.cache,
+                            complete: IndicoUI.Dialogs.Util.progress(),
+                            error: handleAjaxError,
+                            success: function(data) {
+                                if ($.isFunction(ajaxOptions.success)) {
+                                    ajaxOptions.success.call(self.element, data);
+                                }
+
+                                function updateContent(data) {
+                                    if (data) {
+                                        api.set('content.text', ajaxifyForm($(data.html)));
+                                        if (data.js) {
+                                            $('body').append(data.js);
+                                        }
+                                    }
+                                }
+
+                                function ajaxifyForm($form) {
+                                    return $form.ajaxForm({
+                                        url: $form.attr('action'),
```

Comment:
I think this won't work if there's no `action` attribute. Check the ajaxDialog code on how to get the proper URL (search for `xhr.getResponseHeader('X-Indico-URL')`)